tests/integration/RestApi.Https.Tests.ps1

```diff
@@ -116,73 +121,88 @@ Describe 'REST API Requests' {
         }
 
         Receive-Job -Name 'Pode' | Out-Default
-        (Invoke-RestMethod -Uri "$($Endpoint)/close" -Method Get @splatter) | Out-Null
+        # (Invoke-RestMethod -Uri "$($Endpoint)/close" -Method Get @splatter) | Out-Null
+        curl -s -X DELETE "$($Endpoint)/close" -k
         Get-Job -Name 'Pode' | Remove-Job -Force
     }
 
 
     It 'responds back with pong' {
-        $result = Invoke-RestMethod -Uri "$($Endpoint)/ping" -Method Get @splatter
-        $result.Result | Should Be 'Pong'
+        # $result = Invoke-RestMethod -Uri "$($Endpoint)/ping" -Method Get @splatter
+        $result = (curl -s -X GET "$($Endpoint)/ping" -k) | ConvertFrom-Json
+        $result.Result | Should -Be 'Pong'
     }
 
     It 'responds back with 404 for invalid route' {
-        { Invoke-RestMethod -Uri "$($Endpoint)/eek" -Method Get -ErrorAction Stop @splatter } | Should Throw '404'
+        #  { Invoke-RestMethod -Uri "$($Endpoint)/eek" -Method Get -ErrorAction Stop @splatter } | Should -Throw  -ExpectedMessage '*404*'
+        $status_code = (curl.exe -s -o /dev/null -w '%{http_code}' "$Endpoint/eek" -k)
```

Comment:
`curl` instead of `curl.exe` 😉  there's ~5 of these, otherwise it looks like the switch to curl might have resolved it, as the windows job passed :)

Reply:
I already fixed it. 
I need to commit it